packages/ui/src/components/cell/components/cell-icon.native.tsx

```diff
@@ -0,0 +1,7 @@
+import { Box, BoxProps } from '../../box/box.native';
+
+export interface CellIconProps extends BoxProps {}
+
+export function CellIcon(props: CellIconProps) {
+  return <Box {...props} />;
+}
```

Comment:
is this component really needed?

Reply:
I considered removing this as it's right now noop and holds no opinions on its contents, but thought to keep around as we'll likely need to tweak the spacing for the re-designed double icon Avatars that leak out of the square bounding box by design.

<img width="500" alt="Screenshot 2024-12-06 at 18 57 57" src="https://github.com/user-attachments/assets/ef78b809-19ee-4c29-a823-643b07889dcc">